src/service/ui/notification.js

```diff
@@ -45,6 +45,7 @@ const ReplyDialog = GObject.registerClass({
     _init(params) {
         super._init({
             application: Gio.Application.get_default(),
+            title: _('Reply'),
```

Comment:
In this case, `headerbar.title` is set to `params.notification.appName`, which will be the title of the dialog in every way that matters. What's the purpose of also setting it generically, here?